{ 
  "text" : "** DISPUTED **   An issue was discovered in NumPy 1.16.0 and earlier. It uses the pickle Python module unsafely, which allows remote attackers to execute arbitrary code via a crafted serialized object, as demonstrated by a numpy.load call. NOTE: third parties dispute this issue because it is  a behavior that might have legitimate applications in (for example)  loading serialized Python object arrays from trusted and authenticated  sources.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "ThreatActor#d0f531219210b35f7261f3d6049ee44c" ,
    "type" : "ThreatActor" ,
    "subType" : "ThreatActor" ,
    "value" : "remote attacker" ,
    "begin" : 126 ,
    "end" : 142 ,
    "canonicalForm" : "remote attacker" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "f8cac63bd61f61705fa92b1462da0c50" ,
    "type" : "MiscEntity" ,
    "subType" : "code" ,
    "value" : "arbitrary code" ,
    "begin" : 154 ,
    "end" : 168 ,
    "canonicalForm" : "arbitrary code" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [ { 
    "rid" : "R1" ,
    "type" : "execute" ,
    "predicate" : { 
      "begin" : 143 ,
      "end" : 153 ,
      "surfaceForm" : "to execute"
    }
     ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 126 ,
      "end" : 142 ,
      "eid" : "E1" ,
      "canonicalForm" : "remote attacker"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 154 ,
      "end" : 168 ,
      "eid" : "E2" ,
      "canonicalForm" : "arbitrary code"
    }
    ,  ]
  }
  ,  ]
}
